docker/volumeplugin/plugin.go

```diff
@@ -79,70 +77,96 @@ func (d *RancherStorageDriver) init() error {
 }
 
 func (d *RancherStorageDriver) Create(request volume.Request) volume.Response {
-	logrus.Infof("Docker Create request: %v", request)
+	logRequest("create", &request)
+
+	response := volume.Response{}
+	defer logResponse("create", &response)
 
 	if created, err := d.state.IsCreated(request.Name); err != nil {
-		return volErr(err)
+		response.Err = err.Error()
```

Comment:
Example with changes:

`time="2016-11-16T18:18:08Z" level=info msg=create.request name="elated-shannon_etcd_backup_a82d3" options=map[size:499 volumeType:st1]`
`time="2016-11-16T18:18:09Z" level=error msg=create.response error="Failed in create: \nAn error occurred (InvalidParameterValue) when calling the CreateVolume operation: The volume size is invalid for st1 volumes: 499 GiB. st1 volumes must be at least 500 GiB in size. Please specify a volume size above the minimum limit."`